.gitignore

```diff
@@ -7,3 +7,4 @@ sensor-names.toml
 .DS_Store
 .vscode
 /notebooks/data
+*.tar
```

Comment:
This seems a little broad, could you make it a bit more specific, like `*.mijia-homie.tar` instead?